Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,76 @@
-﻿using System;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
-namespace BootCamp.Chapter
+namespace Homework1
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Welcome to the self-confidence crusher! Also known as the BMI Calculator. Please input your information below.");
+            Console.WriteLine();
+
+            // First Person Start.
+            Console.Write("Can the first person please press enter to start!");
+            Console.ReadKey();
+            Console.WriteLine();
+            Console.WriteLine();
+
+            Console.Write("Enter your First Name: ");
+            string firstName = Console.ReadLine();
+            
+            Console.Write("Enter your Last Name: ");
+            string lastName = Console.ReadLine();
+            
+            Console.Write("Enter your Age: ");
+            int age = Convert.ToInt32(Console.ReadLine());
```

Comment:
Human age would always fit to Int16, that's not a big deal, but a thing to consider.

Reply:
If you're being pedantic like that, it'd also fit into a byte.